### PR TITLE
Refine config-cross-recipe script

### PR DESCRIPTION
**Pull request by ffoulkes**
- Break command-line help into Paths and Options sections.

- Adopt recent conventions from make-all.sh.

- Change ES2K_TARGET=YES to TDI_TARGET=ES2K.

- Remove unnecessary echo statements.

- Address shellcheck issues

- Replace old script in top-level directory with symlink to
  revised script in scripts/es2k.

### CODE REVIEW

**Review by vsureshkumarp — APPROVED** [review]
Changes looks good to me.

File: config-cross-recipe.sh [hunk 1]
```diff
@@ -1,156 +0,0 @@
-#!/bin/bash
```
Comment (n-sandeep) [inline config-cross-recipe.sh:1]: now ACC scripts are modified to pick config-cross-recipe.sh from scripts/es2k/config-cross-recipe.sh
Older reference can be removed

Comment (ffoulkes) [reply config-cross-recipe.sh:1]: Cool! Can you approve the PR, so I can merge it?
I'll remove the symlinks in a follow-up commit.

**Review by nupuruttarwar — APPROVED** [review]
LGTM

**Pull request merged**